src/app/browse/br-clothing/br-clothing.component.ts

```diff
@@ -1,16 +1,49 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
+import { item } from '../../Item';
+import { ItemService } from '../../item.service';
+import { Mock } from '../../MockBook';
 
 @Component({
   selector: 'app-br-clothing',
   templateUrl: './br-clothing.component.html',
   styleUrls: ['./br-clothing.component.css']
 })
 export class BrClothingComponent implements OnInit {
+  retrievedItems: item[];
+  items: item[];
 
-  constructor(public router: Router) { }
 
+  constructor(public router: Router, 
+             private ItemService: ItemService) { }
+
+
+  getItems():void{
+  	this.ItemService.getItems()
+  		.subscribe(items => this.retrievedItems = items)
```

Comment:
Here and elsewhere, you do not necessarily need to subscribe to the `getItems()` function. Since the `ItemService` is injected to all of the different `*-item` components and provides and `Observable<item[]>`. The following will allow you to have access to `item$`. 

some-item.component.ts
```typescript
export class SomeItemComponent implements OnInit {
    constructor(private itemSvc: ItemService) { }
}
```

some-item.component.html
```html
<div *ngIf="itemSvc.items$ | async as items">
	<div *ngFor="let item of items">
		... 
	</div>
</div>
```